cli/tests/e2e.test.ts

```diff
@@ -145,7 +145,14 @@ describe("e2e tests", function () {
         useWasm,
       );
       await proveOnChain("0", testProofsDirPath);
-      await verify("0", testTallyFilePath, maciAddresses.maciAddress, pollAddresses.tally, pollAddresses.subsidy);
+      await verify(
+        "0",
+        testTallyFilePath,
+        tallyFileData,
+        maciAddresses.maciAddress,
+        pollAddresses.tally,
+        pollAddresses.subsidy,
+      );
```

Comment:
```suggestion
      await verify(
        "0",
        tallyFileData,
        maciAddresses.maciAddress,
        pollAddresses.tally,
        pollAddresses.subsidy,
      );
```

According [this change](https://github.com/privacy-scaling-explorations/maci/pull/877/files#diff-3b36308f3b5bc6212abedcd7958b17e613a417a8f70e3f759bb6474fd0eb76d2R68-R74), `verify()` won't read tally File

Reply:
this is not going to work, basically tallyFileData will be passed as argument to tallyFilePath causing it to throw an error due to it not finding the file